app/src/main/java/cm/aptoide/pt/social/view/PostAdapter.java

```diff
@@ -128,4 +128,26 @@ private boolean hasUser() {
   public void removePost(Post post) {
     if (posts.remove(post)) notifyDataSetChanged();
   }
+
+  public int getPostPosition(String postId) {
+    int result = -1;
+    for (Post testPost : posts) {
+      if (testPost.getCardId()
+          .equals(postId)) {
+        result = posts.indexOf(testPost);
```

Comment:
why continuing with the loop instead of returning

Reply:
@paquiquinho

Reply:
done